src/server/ua_server_ns0.c

```diff
@@ -649,7 +649,30 @@ UA_Server_initNS0(UA_Server *server) {
 
 #ifdef UA_GENERATED_NAMESPACE_ZERO
     /* Load nodes and references generated from the XML ns0 definition */
-    retVal = namespace0_generated(server);
+    retVal |= namespace0_generated(server);
+
+    /* Remove unsupported nodes */
+    retVal |= UA_Server_deleteNode(server, UA_NODEID_NUMERIC(0, UA_NS0ID_SERVER_SETSUBSCRIPTIONDURABLE), true);
```

Comment:
Why do we need to delete the nodes here?
What does `unsupported` mean?

Are these only unsupported by the CTT, but in general still valid nodes?
Or are these deprecated nodes?

Can you clarify that in the comment please.

I think we should ignore the return value for the deleteNode calls if it is `UA_STATUSCODE_BADNODEIDUNKNOWN`.

Especially if the nodes are deprecated, then future nodesets may just omit them and this code will fail.